sbom/sbom_outputs_test.go

```diff
@@ -17,9 +17,11 @@ type component struct {
 }
 
 type cdxOutput struct {
-	BOMFormat   string `json:"bomFormat"`
-	SpecVersion string `json:"specVersion"`
-	Metadata    struct {
+	BOMFormat    string `json:"bomFormat"`
+	SpecVersion  string `json:"specVersion"`
+	SerialNumber string `json:"serialNumber"`
+	Metadata     struct {
+		Timestamp string `json:"timestamp"`
```

Comment:
Do you think it would be clearer if the serial number and timestamp fields were omitted if empty so they don't show up at all?

Reply:
I don't know I would test for their absence if they were omitted

Reply:
it's alright - tested this out in a build and the fields are omitted anyway.